TheaterSchedule.DAL/Repositories/SettingsRepository.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using TheaterSchedule.DAL.Contexts;
+using TheaterSchedule.DAL.Entities;
+using TheaterSchedule.DAL.Interfaces;
+
+namespace TheaterSchedule.DAL.Repositories
+{
+    class SettingsRepository : ISettingsRepository
+    {
+
+        private TheaterScheduleContext db;
+
+        public SettingsRepository(TheaterScheduleContext context)
+        {
+            this.db = context;
+        }
+
+        //Get
+        public Settings GetSettingsByPhoneId(string SettingsId)
+        {
+            Settings settings = null;
+            settings = db.Account.Select(a => a.AccountNavigation).Where(a => a.SettingsId == SettingsId).SingleOrDefault();
+            return settings;
+        }
+
+        // Post
+        public void CreateNewAccountAndSettingsWithCurrentPhoneId(Settings settings)
```

Comment:
Shouldn't POST method return newly created item in response body?

Reply:
it could do it, but i thought if we use it only once(first entering) and than we go home page, where will be httpGet,so here you get this data.